### DISCUSSION TITLE
How to do CSS's back drop blur filter in Pixi?

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7687] [Q&A]
![image](https://user-images.githubusercontent.com/87828192/128710370-068f0eca-cfaf-4d45-8131-f3e9ce95cc51.png)

I really want to do this in Pixi.
But I can't figure it out..
Thanks in advance.

**Comment by ivanpopelyshev** [comment]
Its super-hard, you probably should assign a big bounty to it! How many other 2d webgl renderers did you see with this feature? 

Actually, you dont have to do anything, its already implemented here: https://pixijs.io/examples/#/plugin-picture/mask-blur.js , you need `@pixi/picture` plugin for that

**Reply by humbleCoder1990** [reply]
Thanks Ivan for always amazing support.
I'm a big PIXI fan in Korea. 
I'm making GUI music application using PIXI, (not game)
I was using React, but finally met very slow performance with many dom elements.
But in PIXI, most stuff was possible with a lot of elements.
thanks a lot. It's best team.

**Reply by ivanpopelyshev** [reply]
Thank you, this means a lot to me!

if pixi blur will be slow for your conditions. you can try Kawase from pixi-filters instead. I have better blur that i didnt publish for 2 years already, just not enough time :(

**Reply by humbleCoder1990** [reply]
![image](https://user-images.githubusercontent.com/87828192/128786701-b29c70b9-4a63-4b07-abc7-e249ba2bb3d4.png)
Thanks Ivan.
I'm following the example on pixijs.io
But I see this error when trying to use 'picture'
Did api get changed?

**Reply by ivanpopelyshev** [reply]
https://github.com/pixijs/picture#usage

All for you, modern react/webpack coders.

**Reply by humbleCoder1990** [reply]
![image](https://user-images.githubusercontent.com/87828192/128787052-b26da78d-f686-4555-a30c-3b9ba7adf3a0.png)

**Reply by ivanpopelyshev** [reply]
did you install it?

**Reply by humbleCoder1990** [reply]
There was 2 pages.

https://www.npmjs.com/package/pixi-picture
https://www.npmjs.com/package/@pixi/picture

I installed old one. (https://www.npmjs.com/package/pixi-picture)
Now I installed new one.

Now I can import via 'import { MaskFilter } from '@pixi/picture'
Thanks ivan. 
I hope It use 'pixi-picture' instead '@pixi/picture'
